types/mongodb/index.d.ts

```diff
@@ -54,13 +54,64 @@ export class MongoClient extends EventEmitter {
     logout(options?: { dbName?: string }): Promise<any>;
     logout(options: { dbName?: string }, callback: MongoCallback<any>): void;
     /** http://mongodb.github.io/node-mongodb-native/3.0/api/MongoClient.html#startSession */
-    startSession(options?: any): ClientSession;
+    startSession(options?: SessionOptions): ClientSession;
 }
 
+/** https://docs.mongodb.com/manual/reference/method/Mongo.startSession/ */
+declare interface SessionOptions {
+    /** https://docs.mongodb.com/manual/core/read-isolation-consistency-recency/#causal-consistency */
+    causalConsistency?: boolean;
+    /**https://docs.mongodb.com/manual/reference/read-concern/#read-concern */
+    readConcern?: ReadConcern;
+    /** https://docs.mongodb.com/manual/reference/read-preference/#read-preference */
+    readPreference?: ReadPreference;
+    /**  */
+    retryWrites?: boolean;
+    /** https://docs.mongodb.com/manual/reference/write-concern/#write-concern */
+    writeConcern?: WriteConcern;
+}
+
+declare type ReadConcern = {
```

Comment:
Can you use `interface` instead of `type` so it's consistent with the other definitions?

Reply:
Yes I can